thrift/src/test/java/com/linecorp/armeria/server/thrift/ThriftServiceTest.java

```diff
@@ -600,6 +604,19 @@ public void testMultipleInheritance() throws Exception {
         assertThat(client2.recv_sort()).containsExactly(NAME_A, NAME_B, NAME_C);
     }
 
+    @Test
+    public void testServiceThrowingException() {
```

Comment:
Could you revert this part now that we know this does not reproduce the problem?

Reply:
Done.